mesonbuild/interpreter.py

```diff
@@ -909,16 +909,25 @@ def get_name(self):
 
 class SubprojectHolder(InterpreterObject, ObjectHolder):
 
-    def __init__(self, subinterpreter):
+    def __init__(self, subinterpreter, name):
         InterpreterObject.__init__(self)
         ObjectHolder.__init__(self, subinterpreter)
+        self.name = name
         self.methods.update({'get_variable': self.get_variable_method,
+                             'found': self.found_method,
                              })
 
+    @noPosargs
+    @permittedKwargs({})
+    def found_method(self, args, kwargs):
+        return bool(self.held_object)
+
     @permittedKwargs({})
     def get_variable_method(self, args, kwargs):
         if len(args) != 1:
             raise InterpreterException('Get_variable takes one argument.')
+        if not self.found_method([], {}):
```

Comment:
nitpicking: I'm not a big fan of calling found_method() like that, I think it would be nicer to have a found() method and make found_mehtod() call found(). I see ExternalProgramHolder does that.

Reply:
@thiblahute I think you forgot to fix this after adding the `found()` method :)